client/src/components/payment/payment-modal.vue

```diff
@@ -1,34 +1,39 @@
 <template>
-  <v-dialog max-width="40%" :value="isVisible" persistent>
-    <v-card class="pa-9">
-      <v-card-text class="h3">Confirm Name Request</v-card-text>
-      <v-card-text class="copy-normal">
+  <v-dialog max-width='40%' :value='isVisible' persistent>
+    <v-card class='pa-9'>
+      <v-card-text class='h3'>
+        Confirm Name Request
+        <countdown-timer :timerName="timerName" colorString="#003366" bgColorString="#efefef" style="float: right"/>
+      </v-card-text>
+      <v-card-text class='copy-normal'>
         <request-details
-          v-bind:applicant="applicant"
-          v-bind:name="name"
-          v-bind:nameChoices="nameChoices"
+          v-bind:applicant='applicant'
+          v-bind:name='name'
+          v-bind:nameChoices='nameChoices'
         />
         <fee-summary
-          v-bind:filingData="[...paymentDetails]"
-          v-bind:fees="[...paymentFees]"
+          v-bind:filingData='[...paymentDetails]'
+          v-bind:fees='[...paymentFees]'
         />
       </v-card-text>
       <v-card-actions>
         <v-spacer></v-spacer>
-        <v-btn @click="confirmPayment" id="payment-pay-btn" class="primary" text>Accept</v-btn>
-        <v-btn @click="hideModal" id="payment-close-btn" class="normal" text>Cancel</v-btn>
+        <v-btn @click='confirmPayment' id='payment-pay-btn' class='primary' text>Accept</v-btn>
+        <v-btn @click='cancelPayment' id='payment-close-btn' class='normal' text>Cancel</v-btn>
       </v-card-actions>
     </v-card>
   </v-dialog>
 </template>
 
-<script lang="ts">
+<script lang='ts'>
```

Comment:
Nothing major but i think within the Vue template itself "" is the standard and elsewhere in this project.

Reply:
True.